client/packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -14,6 +15,7 @@
   "heading.are-you-sure": "Are you sure?",
   "heading.consumption-history": "Consumption History (monthly)",
   "heading.related-documents": "Related documents",
+  "heading.return-items": "Return Items",
```

Comment:
nit: just having a cursory look at other areas, it would seem standard capitalisation might be better for these: "Return item", "Select quantity", etc. (rather than Title Case)